.github/workflows/publish.yml

```diff
@@ -105,4 +107,5 @@ jobs:
       - name: Install package
         shell: bash -el {0}
         run: |
-          conda install -c mindee doctr
+          conda config --set channel_priority strict
+          conda install -c mindee -c pypdfium2-team -c bblanchon -c conda-forge python-doctr
```

Comment:
@frgfm the channel is created automatically the first time it's uploaded correct ? 😅

Reply:
Nope, the channel has to be created along with the token so that we can upload it :)